Tests/GenBank/tls_KDHP01000000.gb

```diff
@@ -0,0 +1,40 @@
+LOCUS       KBUV01000000            3714 rc    DNA     linear   ENV 26-FEB-2018
+DEFINITION  TLS: soil metagenome 16S ribosomal RNA, targeted locus study.
+ACCESSION   KBUV00000000
+VERSION     KBUV00000000.1
+DBLINK      BioProject: PRJNA420556
+            BioSample: SAMN08114915
+KEYWORDS    TLS; Targeted Locus Study.
+SOURCE      soil metagenome
+  ORGANISM  soil metagenome
+            unclassified sequences; metagenomes; ecological metagenomes.
+REFERENCE   1  (bases 1 to 3714)
+  AUTHORS   Buscardo,E., Geml,J., Schmidt,S., Freitas,H., Brandao,H. and
+            Nagy,L.
+  TITLE     Spatio-temporal dynamics of soil bacterial communities in function
+            of Amazon forest phenology
+  JOURNAL   Unpublished
+REFERENCE   2  (bases 1 to 3714)
+  AUTHORS   Buscardo,E., Geml,J., Schmidt,S., Freitas,H., Brandao,H. and
+            Nagy,L.
+  TITLE     Direct Submission
+  JOURNAL   Submitted (22-NOV-2017) Centre for Functional Ecology, University
+            of Coimbra, Calcada Martim de Freitas, Coimbra, Portugal
+COMMENT     The soil metagenome targeted locus study (TLS) project has the
+            project accession KBUV00000000.  This version of the project (01)
+            has the accession number KBUV01000000, and consists of sequences
+            KBUV01000001-KBUV01003714.
+FEATURES             Location/Qualifiers
+     source          1..3714
+                     /organism="soil metagenome"
+                     /mol_type="genomic DNA"
+                     /isolation_source="soil samples in a lowland tropical
+                     evergreen rain forest in Amazonia"
+                     /db_xref="taxon:410658"
+                     /environmental_sample
+                     /country="Brazil: Manaus"
+                     /lat_lon="2.92 S 59.95 W"
+                     /collection_date="2013"
+                     /note="metagenomic"
+TLS         KBUV01000001-KBUV01003714
+//
```

Comment:
Is that exactly as the NCBI did it - missing a new line character after the slash slash?

Reply:
There is no new line here: https://www.ncbi.nlm.nih.gov/nuccore/GAAA00000000.1
Should we add it?

Reply:
https://www.ncbi.nlm.nih.gov/nuccore/GAAA00000000.1 is HTML so no real guide.

Using the send to file menu, I get a trailing new line (as expected) and a blank line:

```
$ hexdump -C  ~/Downloads/sequence.gb  | tail
000009b0  76 69 64 65 64 20 62 79  20 44 72 2e 0a 20 20 20  |vided by Dr..   |
000009c0  20 20 20 20 20 20 20 20  20 20 20 20 20 20 20 20  |                |
000009d0  20 20 43 68 72 69 73 20  41 6d 65 6d 69 79 61 20  |  Chris Amemiya |
000009e0  6f 66 20 42 65 6e 61 72  6f 79 61 20 52 65 73 65  |of Benaroya Rese|
000009f0  61 72 63 68 20 49 6e 73  74 69 74 75 74 65 2c 20  |arch Institute, |
00000a00  53 65 61 74 74 6c 65 22  0a 54 53 41 20 20 20 20  |Seattle".TSA    |
00000a10  20 20 20 20 20 47 41 41  41 30 31 30 30 30 30 30  |     GAAA0100000|
00000a20  31 2d 47 41 41 41 30 31  30 39 33 35 30 37 0a 2f  |1-GAAA01093507./|
00000a30  2f 0a 0a                                          |/..|
00000a33

```